README.md

```diff
@@ -47,6 +47,21 @@ WebNN requires a compatible browser to run, and Windows 11 v21H2 (DML 1.6.0) or
 4. In the drop-down menu, select `Enabled`. 
 5. Relaunch your browser.
 
+#### Running WebNN on NPU
+Running WebNN on NPU requires capable NPU hardware e.g. Intel® AI Boost NPU from Intel® Core™ Ultra processors.
+
+* Google Chrome Canary:
```

Comment:
@reillyeon , please help take a look at the steps of testing WebNN samples on NPU in Chrome Canary. Thanks!